src/charity_base.yaml

```diff
@@ -0,0 +1,36 @@
+---
+# Taken from their website's "What is CharityBase?"
+description: >
+  Charity Base is an API that aggregates data from the 168,000 charities
+  registered in England and Wales.
+# One Project is also licensed under Apache-2.0 and some don't have
+# a license
+license: MIT
+name: CharityBase
+programmingLanguages:
+  - javascript
+  - python
+rating: 2
```

Comment:
Maybe we could add a comment to explain the rating? At least until we get an algorithm.
Here, I'd say that the project is awesome within Britain, but could be made more friendly to contributors and try to scale globally

Reply:
Currently, the ratings don't really mean anything other than a personal gauge ^^'

I agree, that the project itself is great, but considering the lacking documentation and basically non-existing contribution guide (that is also somewhat hidden) would lead me to believe they would get a low score.

I'll add a quick comment and then rebase the PR, if that's okay with you

Reply:
Oh, that's good! Sounds more friendly this way :) Definitely a good opportunity for some folks out there :) 

Go ahead, Thanks!